src/components/greetings/greetings.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { noop } from "../../services/utils";
+
+import createGreetingsLabel from "./greetingsLabel";
+import createBackButton from "./greetingsBackButton";
+
+export default ({ data = {}, onBack = noop } = {}) => {
```

Comment:
files that provide components should be started form capital latter

Reply:
these are erroneous files, I forgot to delete them